spec/unit/recipes/default_spec.rb

```diff
@@ -5,18 +5,19 @@
 # Copyright (c) 2015 Irving Popovetsky, All Rights Reserved.
 
 require 'spec_helper'
+require 'mixlib/versioning' # new chef-ingredient makes heavy use of this library
```

Comment:
It seems weird to me that you have to require this here. Is it not getting required in the right place in `chef_ingredient`? Does there need to be a Gemfile with this included?